example/.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"flutter_twilio_voice","path":"/Users/diego/Documents/HomeTask/flutter_twilio_voice/","dependencies":[]},{"name":"cloud_functions","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/cloud_functions-0.6.0/","dependencies":["firebase_core"]},{"name":"firebase_auth","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_auth-0.18.0+1/","dependencies":["firebase_core"]},{"name":"firebase_core","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_core-0.5.0/","dependencies":[]},{"name":"firebase_messaging","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.0/","dependencies":["firebase_core"]}],"android":[{"name":"flutter_twilio_voice","path":"/Users/diego/Documents/HomeTask/flutter_twilio_voice/","dependencies":[]},{"name":"cloud_functions","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/cloud_functions-0.6.0/","dependencies":["firebase_core"]},{"name":"firebase_auth","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_auth-0.18.0+1/","dependencies":["firebase_core"]},{"name":"firebase_core","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_core-0.5.0/","dependencies":[]},{"name":"firebase_messaging","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.0/","dependencies":["firebase_core"]}],"macos":[{"name":"cloud_functions","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/cloud_functions-0.6.0/","dependencies":["firebase_core"]},{"name":"firebase_auth","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_auth-0.18.0+1/","dependencies":["firebase_core"]},{"name":"firebase_core","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_core-0.5.0/","dependencies":[]}],"linux":[],"windows":[],"web":[{"name":"cloud_functions_web","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/cloud_functions_web-2.0.0/","dependencies":[]},{"name":"firebase_auth_web","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_auth_web-0.3.0+1/","dependencies":[]},{"name":"firebase_core_web","path":"/Users/diego/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_core_web-0.2.0/","dependencies":[]}]},"dependencyGraph":[{"name":"flutter_twilio_voice","dependencies":[]},{"name":"cloud_functions","dependencies":["firebase_core","cloud_functions_web"]},{"name":"cloud_functions_web","dependencies":[]},{"name":"firebase_auth","dependencies":["firebase_core","firebase_auth_web"]},{"name":"firebase_auth_web","dependencies":["firebase_core"]},{"name":"firebase_core","dependencies":["firebase_core_web"]},{"name":"firebase_core_web","dependencies":[]},{"name":"firebase_messaging","dependencies":["firebase_core"]}],"date_created":"2020-10-10 23:40:17.278675","version":"1.22.0"}
```

Comment:
I guess, this file and some others should be added to gitignore

Reply:
Is actually in the example's gitignore, I don't know why its being tracked.